imagehash/__init__.py

```diff
@@ -116,10 +116,10 @@ def average_hash(image, hash_size=8):
 def phash(image, hash_size=32):
```

Comment:
Could you please keep the old function and rename it to "phash_simple"?
Also you mentioned that you looked into what phash.org implements. Would it make sense to have one or more of their algorithms? Obviously there is more than one way to implement a "perceptual hash", so they could be phash_xxx functions.